### DISCUSSION TITLE
How to uninstall it?

### DISCUSSION THREAD

**Discussion by OrionRandD** [pawelsalawa/sqlitestudio#5129] [Q&A]
Could you add info about how to uninstall sqlitestudio as well?
Any opensource software should show installation and de(installation) process...

**Comment by pawelsalawa** [comment]
SQLiteStudio can be uninstalled like any other application on your operating system. Since it is a tool aimed at database professionals and enthusiasts, it is assumed that users are familiar with basic application management on their chosen platform.

**Comment by secondkings2-24** [comment]
I can't install sqlitestudio in anyway on linux to make it work. There are now parts of the program that can't be removed at all. I have never had this issue with another program. It looked like an excellent database program too.

**Reply by secondkings2-24** [reply]
I am using Ubuntu, btw

**Reply by secondkings2-24** [reply]
I install it, but I can't run it or open it. That's why I was trying uninstall and maybe and reinstall it.

**Reply by pawelsalawa** [reply]
So... after you've installed it - how are you trying to start it? What exactly do you do?
